lib/fs/dumbfs.c

```diff
@@ -42,32 +43,125 @@ dumbfs_unmount(void *cookie)
 
 static status_t
 dumbfs_open(
```

Comment:
This function heavily uses spaces for indentation. Be aware that ChaOS tends to minise such things, using mostly tabulations.